src/physics/bomb-jump.s

```diff
@@ -0,0 +1,5 @@
+; Removes check for iframes when bomb jumping
```

Comment:
can you elaborate on this a bit more please? i wouldnt understand this without needing to dig up the pr/issue for context.